pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "navani"
-version = "0.1.5"
+version = "0.1.9"
```

Comment:
Minor suggestion with this, use git tags to set the versions and use something like `setuptools_scm` to set the version dynamically (see the datalab-api package for e.g.)